wgsl/index.bs

```diff
@@ -1753,6 +1753,15 @@ The originating variable for a formal parameter of a function depends on the
 [=call site|call sites=] for the function.
 Different call sites may supply pointers into different originating variables.
 
+If a reference or pointer access is out of bounds, an <dfn noexport>invalid
+memory reference</dfn> is produced.
+[=Load Rule|Loads=] from an invalid reference return either:
```

Comment:
this doesn't include the RBA behavior, but the group consensus was to allow it:
> Values from anywhere within the memory range(s) bound to the buffer (possibly including bytes of memory past the end of the buffer, up to the end of the bound range).

Which effectively means any memory whatsoever (yikes)

Reply:
I will change this to say any memory location of the originating variable.

Reply:
I think this works because the WGSL spec says the struct size is rounded up to the alignment so there may be padding bytes affected.

Reply:
How do you possibly define the "memory location of the originating variable"? I.e. a Vulkan backend is expected to sub-allocate buffers from large memory allocations, in which case you can out-of-bound read one buffer in WebGPU and get the values from another buffer.

Reply:
The way I read the RBA text in Vulkan is that the behaviour is restricted to the buffer. So I would think it can only write into the next buffer if it is placed in the padding of the previous buffer which is intended to be captured by all memory locations of the variable.

Reply:
Ok, thanks for correction. I recall previous discussion a while ago concluding that RBA can only guarantee accesses to stay within memory, not just within the bound range, but it's possible this was a misunderstanding.